bin/check-s3-bucket-visibility.rb

```diff
@@ -0,0 +1,131 @@
+#! /usr/bin/env ruby
+#
+# check-s3-bucket-visibility
+#
+# DESCRIPTION:
+#   This plugin checks a bucket for website configuration and bucket policy.
+#   It alerts if the bucket has a website configuration, or a policy that has
+#   Get or List actions.
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk
+#   gem: sensu-plugin
+#
+# USAGE:
+#   ./check-s3-bucket-visibility.rb --bucket-name mybucket --aws-region eu-west-1
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright (c) 2015, Olivier Bazoud and Ricky Hussmann,
+#     olivier.bazoud@gmail.com, ricky.hussmann@gmail.com
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class CheckS3Bucket < Sensu::Plugin::Check::CLI
+  option :aws_access_key,
+         short: '-a AWS_ACCESS_KEY',
+         long: '--aws-access-key AWS_ACCESS_KEY',
+         description: "AWS Access Key. Either set ENV['AWS_ACCESS_KEY'] or provide it as an option",
+         default: ENV['AWS_ACCESS_KEY']
+
+  option :aws_secret_access_key,
+         short: '-k AWS_SECRET_KEY',
+         long: '--aws-secret-access-key AWS_SECRET_KEY',
+         description: "AWS Secret Access Key. Either set ENV['AWS_SECRET_KEY'] or provide it as an option",
+         default: ENV['AWS_SECRET_KEY']
+
+  option :aws_region,
+         short: '-r AWS_REGION',
+         long: '--aws-region REGION',
+         description: 'AWS Region (defaults to us-east-1).',
+         default: 'us-east-1'
+
+  option :aws_session_token,
+         short: '-t AWS_SESSION_TOKEN',
+         long: '--aws-session-token TOKEN',
+         description: 'AWS Session Token',
+         default: ENV['AWS_SESSION_TOKEN']
+
+  option :bucket_name,
+         short: '-b BUCKET_NAME',
+         long: '--bucket-name',
+         description: 'The name of the S3 bucket to check',
+         required: true
+
+  option :use_iam_role,
+         short: '-u',
+         long: '--use-iam',
+         description: 'Use IAM role authenticiation. Instance must have IAM role assigned for this to work'
+
+  def aws_config
+    { access_key_id: config[:aws_access_key],
+      secret_access_key: config[:aws_secret_access_key],
+      session_token: config[:session_token],
+      region: config[:aws_region] }
+  end
```

Comment:
instead of this can you `include Common` and also `require sensu-plugins-aws`. That way it's consistent with how the other checks here are configured. See https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/bin/check-alb-target-group-health.rb#L38 for example.